templates/invoices/show.html

```diff
@@ -6,13 +6,11 @@
 {% endblock %}
 
 {% block action %}
-{% comment "later" %}
-    {% if user.username == invoice.owner.user.username %}
-        <a href="{% url 'invoices-edit' invoice.id %}" class="primary-action">
+    {% if user.username == invoice.owner.user.username or user.profile.firmatecknare %} <!--Doesn't seem to work, need more checks-->
```

Comment:
```suggestion
    {% if user.username == invoice.owner.user.username or user.profile.firmatecknare() %}
```
Just guessing, but does this work?

Reply:
Nope, the original seems to work, and the suggestion doesn't 
```
TemplateSyntaxError at /invoices/1/
Could not parse the remainder: '()' from 'user.profile.firmatecknare()'
```
I don't know why I've commented that it shouldn't work.

Reply:
Oh alright!